File: Simple/Expenses.java
import java.util.*;

public class Expenses {
    public static void main(String[] args) {

        Scanner scan = new Scanner(System.in);

        System.out.print("Meals: ");
        int meals = scan.nextInt();
        System.out.print("Transport: ");
        int transport = scan.nextInt();
        System.out.print("Savings: ");
        int saving = scan.nextInt();

        int monthlyTotal = (meals + transport + saving) * 30;
        System.out.println("Monthly Expenses: RM" + monthlyTotal);
    }

}

File: Simple/Expenses.cpp
#include <iostream>

int main(int argc, char const *argv[])
{
    double meal;
    double transport;
    double saving;

    std::cout<<"Daily Meals Expenses: RM";
    std::cin>>meal;
    std::cout<<"Daily Transportation Expenses: RM";
    std::cin>>transport;
    std::cout<<"Daily Savings: RM";
    std::cin>>saving;

    std::cout<<"Your Monthly Expenses is RM"<<(meal+transport+saving)*30;

    return 0;
}
